app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
 
+cache_dict = {}
+
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def inner(*args, **kwargs) -> int:
```

Comment:
Indicate that inner could return `Any` type in the return type annotation, not only `int`